tools/packaging/common/envoy_bootstrap.json

```diff
@@ -550,6 +550,19 @@
              "port_value": {{ .envoy_status_port }}
            }
         },
+        {{- if .dual_stack }}
+        "additionalAddresses": [
```

Comment:
should the metrics endpoint above also listen on additional address?

Reply:
The current PR is only for /healthz/ready, with integration tests added for the same. I intend to add integration tests for metrics in a separate PR, and at that time, the bugs related to that can be fixed. Don't want to put everything into a single PR

Reply:
sounds great, thanks